shared/devices/device-page/index.render.js.flow

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+
+export type TimelineItem = {
+  type: 'LastUsed' | 'Added' | 'Revoked',
+  desc: string
+}
+
+export type BannerItem = {
+  type: 'OutOfDate' | 'WillUnlock',
+  desc: string
+}
+
+export type Props = {
+  type: 'mobile' | 'desktop' | 'paperKey',
```

Comment:
Cool. This is purely dumb so it'll change. I'll update in the morning

On Monday, April 25, 2016, Chris Ball <notifications@github.com> wrote:

> In shared/devices/device-page/index.render.js.flow
> <https://github.com/keybase/client/pull/2692#discussion_r61005656>:
>
> > @@ -0,0 +1,24 @@
> > +/* @flow */
> > +
> > +export type TimelineItem = {
> > +  type: 'LastUsed' | 'Added' | 'Revoked',
> > +  desc: string
> > +}
> > +
> > +export type BannerItem = {
> > +  type: 'OutOfDate' | 'WillUnlock',
> > +  desc: string
> > +}
> > +
> > +export type Props = {
> > +  type: 'mobile' | 'desktop' | 'paperKey',
>
> My service seems to send paper keys as type "backup"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2692/files/c8b8fa225f3b8f003d507349905f145954f8e25c#r61005656>
>